src/client/scripts/game/chess/game.js

```diff
@@ -82,6 +82,7 @@ const game = (function(){
         if (input.isKeyDown('tab')) guipause.callback_TogglePointers();
         if (input.isKeyDown('r')) piecesmodel.regenModel(game.getGamefile(), options.getPieceRegenColorArgs(), true);
         if (input.isKeyDown('n')) options.toggleNavigationBar();
+        if (input.isKeyDown('c')) premove.clearPremoves();
```

Comment:
Should we stick to right-click cancelling all premoves?

<img width="926" alt="Screen Shot 2024-07-22 at 12 16 06 PM" src="https://github.com/user-attachments/assets/37b1a03d-caff-4654-a87d-5b1c815c4ee0">

selection.js line 72